streamz/tests/test_kafka.py

```diff
@@ -122,6 +122,7 @@ def test_from_kafka():
         stream = Stream.from_kafka([TOPIC], ARGS, asynchronous=True)
         out = stream.sink_to_list()
         stream.start()
+        sleep(5)
```

Comment:
Why this sleep?

Reply:
This can probably be removed. I'll take a closer look